### PR TITLE
feat: Uploader UI Complete

### PR DESCRIPTION
**Pull request by topboyasante**
- Built the UI for the Uploader with the `react-dropzone` library

### CODE REVIEW

File: src/components/ui/uploader.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  ));
+
+  return (
+    <section className="w-full p-5">
+      <div
+        {...getRootProps({ className: "dropzone" })}
+        className="flex w-full flex-col items-center justify-center  rounded-md border border-dashed border-border px-5 py-10"
+      >
+        <input {...getInputProps()} />
+        <p className="text-center text-[#777777]">
```
Comment (tedspare) [inline src/components/ui/uploader.tsx:37]: Use `text-neutral-500` in favor of hex code

File: src/components/ui/uploader.tsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+          Drag and drop some files here, or click the button to select files
```
Comment (tedspare) [inline src/components/ui/uploader.tsx:38]: `Drag and drop some files here, or click the button to select files` → `Drop files here or click to select`

File: src/components/ui/uploader.tsx [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    // Disable click and keydown behavior
+    noClick: true,
+    noKeyboard: true,
+    onDrop,
+  });
+
+  //Shows a list of files that have been accepted
+  const files = acceptedFiles.map((file: FileWithPath) => (
+    <li key={file.path}>
+      {file.path} - {file.size} bytes
```
Comment (tedspare) [inline src/components/ui/uploader.tsx:26]: Could you format bytes to kb, Mb, etc? 

Something like:

```ts
function getFileSizeSuffix(bytes: number): string {
  const suffixes = ['','K','M','G','T'];
  const magnitude = Math.floor(Math.log2(bytes) / 10);
  const suffix = suffixes[magnitude] + "B";
  return suffix;
}
```

eg. `getFileSizeSuffix(1024)` → `"KB"`

**Review by tedspare — APPROVED** [review]
LGTM, welcome to the repo Nana!! 💪 

Tested locally, works and looks great:

image

**Comment by topboyasante** [comment]
Hi! thanks! I've made the changes

**Pull request merged**